index.js

```diff
@@ -1,27 +1,25 @@
 'use strict';
-var arrify = require('arrify');
-var devices = require('./data.json').devices;
+const arrify = require('arrify');
+const devices = require('./data.json').devices;
 
-module.exports = function (items) {
+module.exports = items => {
 	items = arrify(items);
```

Comment:
Not sure why we arrify it here. The docs only documents an array. I think we can drop this and rather do an array type check.